src/paperless_mail/tests/test_api.py

```diff
@@ -166,6 +170,110 @@ def test_update_mail_account(self):
         self.assertEqual(returned_account2.name, "Updated Name 2")
         self.assertEqual(returned_account2.password, "123xyz")
 
+    def test_mail_account_test_fail(self):
+        """
+        GIVEN:
+            - Errnoeous mail account details
+        WHEN:
+            - API call is made to test account
+        THEN:
+            - API returns 400 bad request
+        """
+        self.bogus_mailbox = BogusMailBox()
+
+        patcher = mock.patch("paperless_mail.mail.MailBox")
```

Comment:
Using this as a context manager in both places will probably resolve the failures

Reply:
cheers, think I got it?